### PR TITLE
add Tag method to user

### PR DESCRIPTION
**Pull request by ammario**
Simplifies process of tagging a user in messages

### CODE REVIEW

File: user.go [hunk 1]
```diff
@@ -17,3 +19,8 @@ type User struct {
 func (u *User) String() string {
 	return u.Username + "#" + u.Discriminator
 }
+
+//Tag return a string which tags the user
+func (u *User) Tag() string {
+	return fmt.Sprintf("<@%v>", u.ID)
```
Comment (iopred) [inline user.go]: %s for strings.

**Comment by iopred** [comment]
Sorry to be a butt, there's a couple of things we should do here:

a) We should probably call this `Mention()`
b) We do string concatenation in `String()` above, we should be consistent here instead of formatting.

**Comment by diraven** [comment]
It's still Sprintf, not concatenation.

**Comment by diraven** [comment]
Also in https://github.com/bwmarrin/discordgo/releases it still says function name is "Tag".

**Pull request merged**